test/unit/client.test.ts

```diff
@@ -851,6 +847,91 @@ describe('Client', () => {
         conversationId: 'test-convo-id-123'
       })
     })
+
+    it('will update the SDK defaults if the actualResolution differs from the requested', async () => {
+      sdk = constructSdk();
+      const stream1 = new MockStream() as any as MediaStream;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9789077)
```suggestion
      const stream1 = new MockStream() as unknown as MediaStream;
```